__tests__/app.test.js

```diff
@@ -92,6 +92,24 @@ describe('GET /api/articles', () => {
             })
         })
     })
+    test('should return an array of only articles of a certain specified topic - cats', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body}) => {
+            body.articles.forEach((article)=>{
+                expect(article.topic).toBe('cats')
+        })
+    })
+    })
+    test('should return a 400, bad request when an invalid topic query is entered - dogs', () => {
+        return request(app)
+        .get('/api/articles?topic=dogs')
+        .expect(400)
+        .then((response) => {
+            expect(response.body.message).toBe('Invalid topic query entered: dogs')
+        })
+    })
```

Comment:
what happens if they search for the topic 'paper' which has no articles but _is_ a valid topic?

Reply:
Lovely implementation!